.vscode/cspell.json

```diff
@@ -69,6 +69,7 @@
     "fhir",
     "finalizer",
     "finalizers",
+    "Hnsw",
```

Comment:
[nit] Most of the words here are in lowercase, consider moving this to lower case too.